lib/UA.js

```diff
@@ -132,14 +133,14 @@ var UA = function(uaString) {
 
 			// Map to different family with constant version
 			} else if (Array.isArray(aliases[this.ua.family]) && aliases[this.ua.family].length >= 2) {
-				a = aliases[this.ua.family];
+				const a = aliases[this.ua.family];
 				this.ua = new useragent.Agent(a[0], a[1], (a[2] || 0), (a[3] || 0));
 
 			// Map to different family with per-version mapping
 			} else if (typeof aliases[this.ua.family] === 'object') {
-				for (semver in aliases[this.ua.family]) {
-					if (this.ua.satisfies(semver) && Array.isArray(aliases[this.ua.family][semver])) {
-						a = aliases[this.ua.family][semver];
+				for (const semverExpr in aliases[this.ua.family]) {
```

Comment:
I'm not sure if this will work – eslint incorrectly reports that this isn't changed and so should be a constant. The following prints "foo" three times anyway:

```js
const obj = { foo: 'bar', bar: 'baz', baz: 'qux' };
for (const prop in obj) { console.log(prop); }
```

Worth investigating?